web/src/lib/utils/asset-utils.ts

```diff
@@ -25,28 +25,13 @@ import { get } from 'svelte/store';
 import { handleError } from './handle-error';
 
 export const addAssetsToAlbum = async (albumId: string, assetIds: string[]) => {
-  const result = await addAssets({
+  await addAssets({
     id: albumId,
     bulkIdsDto: {
       ids: assetIds,
     },
     key: getKey(),
   });
-  const count = result.filter(({ success }) => success).length;
-  notificationController.show({
-    type: NotificationType.Info,
-    timeout: 5000,
-    message:
-      count > 0
-        ? `Added ${count} asset${count === 1 ? '' : 's'} to the album`
```

Comment:
As Daniel mentioned, this seems to remove the current functionality where we send a notification when assets already existing on the server are added to an album (not uploaded to an album). We would like to retain that notification but suppress (or group it) during the upload case.

Reply:
Ah I get you, sorry I thought he was referring to uploading, apologies. Will rectify this evening.

Reply:
I've just  double checked, and the cummulative total assets notification shown in my gif appears for me when adding pre-existing assets to an album. It appears both when adding them during creation of an album, and when added after the album has been created.

![existing-assets-albums](https://github.com/immich-app/immich/assets/91856576/9d080024-7d63-4f1a-b90e-3ff8b2ee2c60)

Reply:
Wtf?

Reply:
?

Reply:
I think probably needs refactoring wherever the notifications are used so that the Notification[] is at most 3 items long, with each of the NotificationType enums being one of the 3 indexes in the array.

Then have it display the count of each NotificationType in their own notification.

if that makes sense?

Reply:
You can drag and drop directly into the album and that should trigger it.

Reply:
Yes, that's what's shown in the GIFs.